server/services/SsotToRobotParsing/SsotToRobotParser.js

```diff
@@ -2,35 +2,64 @@
  * @category Client
  * @module
  */
```

Comment:
Really like the adjustments you made to this file! Very clean

Reply:
thanks - please resolve the conversation